### PR TITLE
[762] New checkbox field to hide Bio for Affiliate Managers

### PR DESCRIPTION
**Pull request by rlaco-qualityunit**
**Changes proposed in this Pull Request**
New checkbox field to hide Bio for Affiliate Managers was added

**Testing instructions**
Test hide functionality for Bio section in Affiliate Manager

**Checklist**
- [x] My code follows the style guidelines of this project
- [x] My commits follow the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

Close QualityUnit/web-issues#762

### CODE REVIEW

File: taxonomy-ms_directory_affiliate_manager.php [hunk 1]
```diff
@@ -87,9 +87,12 @@
 				</ul>
 			</div>
 			<div class="Directory__blocks">
-				<h2 class="Directory__blocks__title"><span><?php _e( 'Affiliate manager\'s bio', 'ms' ); ?></span></h2>
-					<p><?= $aff_desc; // @codingStandardsIgnoreLine ?></p>
-
+				<?php
+				if ( false == $bio_hide ) {
+					?>
+					<h2 class="Directory__blocks__title"><span><?php _e( 'Affiliate manager\'s bio', 'ms' ); ?></span></h2>
```
Comment (jremen) [inline taxonomy-ms_directory_affiliate_manager.php:93]: is having <span> here necessary?

File: taxonomy-ms_directory_affiliate_manager.php [hunk 2]
```diff
@@ -87,9 +87,12 @@
...
+                    <p><?= $aff_desc; // @codingStandardsIgnoreLine ?></p>
+				<?php } ?>
 				<h2 class="Directory__blocks__title"><span><?php _e( 'Geographical focus', 'ms' ); ?></span></h2>
```
Comment (jremen) [inline taxonomy-ms_directory_affiliate_manager.php:96]: Same, <span>

**Review by jremen — COMMENTED** [review]
Check <span> tags and eventually remove them, or at least, give them some class if used

**Comment by rlaco-qualityunit** [comment]
> Check tags and eventually remove them, or at least, give them some class if used

@jremen Fixed

**Review by jremen — APPROVED** [review]

**Pull request merged**